recipes-bsp/opensbi/files/ae350-ax45mp/0002-Enable-cache-for-opensbi-jump-mode.patch

```diff
@@ -0,0 +1,26 @@
+From 325328f4204b40b1fcc8db3b46c7c8805710d21c Mon Sep 17 00:00:00 2001
+From: Yu Chien Peter Lin <peterlin@andestech.com>
+Date: Thu, 30 Dec 2021 08:47:34 +0800
+Subject: [PATCH] Enable cache for opensbi jump mode
```

Comment:
This should also be upstreamed, and the status of the patch should be included. See: https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Importing_from_Elsewhere